CoEpi/ui/healthquiz/HealthQuizViewController.swift

```diff
@@ -0,0 +1,80 @@
+import UIKit
+import RxSwift
+import RxCocoa
+
+class HealthQuizViewController: UIViewController {
+    private let viewModel: HealthQuizViewModel
+    private let dataSource: HealthQuizQuestionsDataSource = HealthQuizQuestionsDataSource()
+
+    @IBOutlet weak var questionList: UITableView!
+    
+    private let disposeBag = DisposeBag()
+    
+    init(viewModel: HealthQuizViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: String(describing: Self.self), bundle: nil)
+        title = "My Health"
+        
+        dataSource.onChecked = { [weak viewModel] (question, idx) in
+            viewModel?.handleAnswer(question: question, idx: idx)
+        }
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    @IBAction func submit(_ sender: UIButton) {
+        viewModel.onTapSubmit()
+    }
+
+    override func viewDidLoad() {
+        questionList.register(cellClass: QuestionCell.self)
+
+        // hack to get around https://github.com/RxSwiftCommunity/RxDataSources/issues/331
+        DispatchQueue.main.async { [weak self] in
```

Comment:
If it's just showing the warning, let's not use this.

Just for completeness - If we wanted to keep this, I'd have suggested to move the bindings into a function method (e.g. `bind()`). Then it becomes possible to put the `DispatchQueue.main.async` in an extensions and "hide" the hack in the view controllers.